services/ows_refactored/prod_af_ows_root_cfg.py

```diff
@@ -233,7 +233,7 @@
                     "abstract": """Agriculture""",
                     "layers": [
                         {
-                            "include": "ows_refactored.agriculture.ows_crop_mask_cfg.layer",
+                            "include": "ows_refactored.agriculture.ows_crop_mask_cfg.layers",
```

Comment:
This makes the layer available in prod, so you probably need to update the inventory for prod as well.